lib/todo_lint/cli.rb

```diff
@@ -6,9 +8,10 @@ class Cli
     # @example
     #   Cli.new([".rb", ".js"])
     # @api public
-    def initialize(extensions = [])
+    def initialize(args)
+      @options = TodoLint::Options.new.parse(args)
```

Comment:
see if you can omit the `TodoLint::` -- we're in the context of the namespace, so we shouldn't need to reference it